src/tests/ftest/util/telemetry_utils.py

```diff
@@ -452,25 +450,62 @@ def get_select_metrics(self, metric_names):
                         data[host][name] = metric["value"]
         return data
 
-    def get_container_metrics(self):
-        """Get the container telemetry metrics.
-
-        Returns:
-            dict: dictionary of dictionaries of container metric names and
-                values per server host key
-
-        """
-        return self.get_select_metrics(self.ENGINE_CONTAINER_METRICS)
-
     def get_io_metrics(self):
-        """Get the I/O telemetry metrics.
+        """Get the io telemetry metrics.
 
         Returns:
-            dict: dictionary of dictionaries of io metric names and
+            dict: dictionary of dictionaries of container metric names and
```

Comment:
It looks like the data dictionary is big and complex. You might want to provide some sample.

Reply:
Yes. I will add a sample telemetry io matrics to the next jira commit.